### PR TITLE
Add email forwarding functionality

### PR DESCRIPTION
**Pull request by luddebubb**
Added a function to send an email with an attachment called `invoice.pdf`.

- The email is currently sent from `sonywalkmanwmfx199@gmail.com`.
- The API controller assumes a request body of `form-data` type containing two key-value pairs.
   - "file": The PDF-file of the invoice
   - "to": The mail address to forward the email to

_<sub>This may or may not work on other devices</sub>_

### CODE REVIEW

File: backend/src/main/java/net/finman/service/EmailServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    private JavaMailSender emailSender;
+
+    @Override
+    public void sendEmailWithAttachment(String to, String subject, String text, InputStreamSource attachment)
+            throws EmailNotSentException {
+        try {
+            MimeMessage message = emailSender.createMimeMessage();
+            MimeMessageHelper helper = new MimeMessageHelper(message, true);
+
+            helper.setFrom("alexIsNoob@noobmail.com");
```
Comment (Puh00) [inline backend/src/main/java/net/finman/service/EmailServiceImpl.java]: is the email sent from sonywalkmanwmfx199@gmail.com or alexIsNoob@noobmail.com?

Comment (yenanw) [reply backend/src/main/java/net/finman/service/EmailServiceImpl.java]: apparently sender overrides by the username specified in application.properties when using google's SMTP server

Comment (yenanw) [reply backend/src/main/java/net/finman/service/EmailServiceImpl.java]: so we just left this here as an easter egg

Comment (Puh00) [reply backend/src/main/java/net/finman/service/EmailServiceImpl.java]: so line 27 can be removed?

Comment (yenanw) [reply backend/src/main/java/net/finman/service/EmailServiceImpl.java]: ~idk didnt test~

tested and it works

Comment (Puh00) [reply backend/src/main/java/net/finman/service/EmailServiceImpl.java]: otherwise kind of misleading

Comment (yenanw) [reply backend/src/main/java/net/finman/service/EmailServiceImpl.java]: u just dont want to be a noob

**Review by Puh00 — APPROVED** [review]
lgtm

**Pull request merged**